lib/cfa/augeas_parser.rb

```diff
@@ -19,6 +36,7 @@ def add(value, placer = AppendPlacer.new)
       element = placer.new_element(@tree)
       element[:key] = augeas_name
       element[:value] = value
+      # FIXME: load_collection missing here
```

Comment:
but I agree that it do not have so much testing as rest where I do it more intensively